syft/core/hooks/torch/hook.py

```diff
@@ -550,15 +427,15 @@ def _hook_torch_module(self):
                 setattr(torch, 'old_{}'.format(attr), lit)
                 setattr(torch, attr, new_attr)
 
-    # ######## END torch module FUNCTION hooking #########
+    # ######## END GENERIC method/function hooking logic #########
     # ######## BEGIN torch TENSOR hooking #########
 
     def _hook_tensor(self, tensor_type):
         """Overloading a given tensor_type"""
         # Overload 'special' methods here
-        self._hook_tensor___init__(tensor_type)
+        # self._hook_tensor___init__(tensor_type)
         # self.hook_tensor___del__(tensor_type)
```

Comment:
Why is this commented out?  If it's no longer relevant, it should be removed.

Reply:
Done. Removing the __del__ one too since it's not getting used yet. (that's still a TODO)